src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -259,17 +261,13 @@ public void Dispose()
 
         internal async Task HandleValidateWindowsAccountRequest(
             ValidateWindowsAccountRequestParams parameters,
-            RequestContext<ValidateWindowsAccountResult> requestContext)
+            RequestContext<bool> requestContext)
         {
             var domainUserRegex = new Regex(@"^[A-Za-z0-9\\\._-]{7,}$");
```

Comment:
This should use the ValidateWindowsDomainUsername method right? And that would probably be best to return the result so you can get the username/domain from the match results directly instead of parsing the string below.